cypress/integration/Authentication/emailCheck_spec.js

```diff
@@ -0,0 +1,33 @@
+const token = "token";
+const userDetails = {
+    email: "maggie@summit.com",
+    first_name: "Maggie",
+    id: 1,
+    is_staff: true,
+    last_name: "Huang",
+};
+const fakeDetails = {
+    email: "iamnotreal@summit.com",
+    first_name: "fake",
+    id: 2,
+    is_staff: true,
+    last_name: "person",
+};
+
+describe("emailCheck_spec", () => {
+    it("loads email check page", () => {
+        cy.visit("");
+        cy.get("[data-cy=nextButton]");
+        cy.get("[data-cy=createAccountButton]")
+    });
+    it("checks invalid user", () => {
+        cy.get("[data-cy=emailField]").fastType(fakeDetails.email);
+        cy.get("[data-cy=nextButton]").click();
+        cy.get(".MuiFormHelperText-root").contains("Sorry, we couldn't find a user for that email")
```

Comment:
Don't use Mui class here, if MUI ever decides to change their class it'll break this test. Find where .MuiFormHelperText-root exists using react's dev tools and add a data-cy attribute to the element. If it does not exist, add it to the parent class and traverse the Virtual DOM to find it. Also, same comment as above. Instead of using contains, use should assertion.